rs/nns/governance/canister/canister.rs

```diff
@@ -736,6 +736,17 @@ fn update_neuron(neuron: Neuron) -> Option<GovernanceError> {
         .map(GovernanceError::from)
 }
 
+#[cfg(feature = "test")]
+#[update]
+/// Internal method for calling update_neuron.
+fn adopt_proposal(proposal_id: ProposalIdProto) -> Option<GovernanceError> {
+    debug_log("adopt_proposal");
+    governance_mut()
+        .adopt_proposal(proposal_id)
+        .err()
+        .map(GovernanceError::from)
```

Comment:
```suggestion
        .map_err(GovernanceError::from)
```

Ofc, this is not equivalent. I'm ALSO suggesting to change the return type.

Reply:
Let's follow the local convention (as in `update_neuron`, another test-only function).